disputes.md

```diff
@@ -75,36 +73,49 @@ regardless of the justification.  When humans solve their problems by
-including but not limited to:
-
-1. Malware (that is, a package designed to exploit or harm the machine
-   on which it is installed).
-2. Violations of copyright or licenses (for example, cloning an
-   MIT-licensed program, and then removing or changing the copyright
-   and license statement).
-3. Illegal content.
-4. "Squatting" on a package name that you *plan* to use, but aren't
-   actually using.  Sorry, I don't care how great the name is, or how
-   perfect a fit it is for the thing that someday might happen.  If
-   someone wants to use it today, and you're just taking up space with
-   an empty tarball, you're going to be evicted.
-5. Putting empty packages in the registry.  Packages must have SOME
-   functionality.  It can be silly, but it can't be *nothing*.  (See
-   also: squatting.)
-6. Doing weird things with the registry, like using it as your own
-   personal application database or otherwise putting non-packagey
-   things into it.
-7. Other things forbidden by the npm
-   [Code of Conduct](https://www.npmjs.com/policies/conduct)
-   such as hateful language, pornographic content, or harassment.
-
-If you see bad behavior like this, please report it to <abuse@npmjs.com>
-right away.  **You are never expected to resolve abusive behavior on
-your own.**  We are here to help.
+## Squatting
+
+It is against npm's
+[Terms of Use](https://www.npmjs.com/policies/open-source-terms#acceptable-content)
+to publish a package, register a user name or an organization name
+simply for the purposes of reserving it for future use. We handle
+squatting via the dispute process in order to give the current owner
+a chance to make genuine use of the name.
+
+We do not pro-actively scan the registry for squatted packages, so
+the fact that a name is in use does not mean we consider it valid.
+The standards for what we consider squatting depend on what is being
+squatted:
+
+### Packages
+
+Package names are considered squatted if the package has no genuine
+function. This judgement is made by a human, and attempts to "game"
+squatting by making pseudo-functional packages will increase, not
+decrease, the likelihood that we will transfer the package to a user
+who requests it.
+
+### Organizations
+
+Organization names are considered squatted if there are no packages
+published within a reasonable time. If an organization is a paid
+organization, it may have private packages that are invisible to
+third parties. For privacy reasons, we cannot reveal whether or not
+an organization has private packages, so a paid organization will
+never be considered squatted.
+
+Unlike users, organizations do not have an "owner" or published
+email address. If you believe an organization is squatted, email
+<support@npmjs.com> directly and we will contact the organization
+owner on your behalf.
+
+### User names
+
+We are extremely unlikely to transfer control of a user name, as it
+is totally valid to be an npm user and never publish and packages.
```

Comment:
"... for instance, you might be part of an organization or need read-only to private packages".